src/_tests/integration/cuj2.integration-test.js

```diff
@@ -0,0 +1,34 @@
+const { click, findText, waitForNoSpinners, signIntoTerra } = require('./integration-utils')
+const { firecloud } = require('./fire-cloud-utils.js')
+
+
+const workflowName = 'haplotypecaller-gvcf-gatk4'
+
+test.skip('integration', async () => {
+  await page.goto('localhost:3000')
+  await click(page, 'View Examples')
+  await signIntoTerra(page)
+  await findText(page, 'GATK4 example workspaces')
+  await click(page, 'code & workflows')
+  await click(page, workflowName)
+
+  await firecloud.signIntoFirecloud(page)
+  await waitForNoSpinners(page)
+  await findText(page, workflowName)
+  await firecloud.click(page, 'Export to Workspace...')
+  await waitForNoSpinners(page)
+  await findText(page, `gatk/${workflowName}-configured`)
+  await firecloud.click(page, `gatk-${workflowName}-configured-1-link`)
```

Comment:
Combined these two lines and instead used the visible text for selection. The only click now that does not use the visible text is the click of the export button in the modal since there are two buttons then (one in modal one outside) with the same visible text so used the test-id